addons/necropost-highlighter/userscript.js

```diff
@@ -0,0 +1,294 @@
+/**
+ * @author RT_Borg
+ * 
+ * Necropost Highlighter
+ * 
+ * Uses topic id as a proxy for age, which doesn't require loading other pages
+ * to read actual dates.
+ *
+ * The number of new topics per month on Scratch forums is fairly stable over time.
+ * Since the topic interval doesn't need to be precise to determine a likely necropost,
+ * this ends up working very well.
+ *
+ * Inspects topic ids of all posts on the page and highlights any topics significantly
+ * older than surrounding posts.
+ * 
+ * Works with:
+ *   - Addon: Website dark mode and customizable colors
+ *   - Addon: Infinite scrolling
+ *   - Standard and Mobile websites
+ */
```

Comment:
Please remove this large comment - It makes the userscript load slower, since network requests are a thing

Reply:
Thanks. As I said, I'm brand new to javascript. I haven't worked in a language before where comments are a detriment. I guess I just imagined a pipeline that reduced files before transit. I stripped everything but "Uses topic id as a proxy for age, which doesn't require loading other pages to read actual dates." If someone other than me needs to alter this in the future, that much seems necessary to orient them to what's going on.

Reply:
@Weredime Actually that's not true, userscripts are stored on disk, load time does not matter. However, @RT-Borg I do believe that info is better served on inline comments or just some PR comment (for addon testing data).